docs/components/radio.md

```
@@ -2,23 +2,31 @@
 import FwbRadioExampleInline from './radio/examples/FwbRadioExampleInline.vue'
 import FwbRadioExampleLink from './radio/examples/FwbRadioExampleLink.vue'
 import FwbRadioExampleList from './radio/examples/FwbRadioExampleList.vue'
 import FwbRadioExampleListHorizontal from './radio/examples/FwbRadioExampleListHorizontal.vue'
+import FwbRadioExampleStyling from './radio/examples/FwbRadioExampleStyling.vue'
+import FwbRadioExampleValidation from './radio/examples/FwbRadioExampleValidation.vue'
 </script>
 
-# Vue Toggle Radio - Flowbite
+# Vue Radio - Flowbite
 
 #### Get started with the radio component to let the user choose a single option from multiple options in the form of a circle based on multiple styles and colors
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)